packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -700,6 +700,38 @@ void main() {
         await controller.pause();
       });
 
+      testWidgets('onScrubbing is called when the controller seeks',
```

Comment:
You've added this method on the `VideoProgressIndicator` as well as the `VideoScrubber` but there's only a test here for the first.